src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/config/AuthConfig.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.reform.authorisation.ServiceAuthorisationApi;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGeneratorFactory;
+
+@Configuration
+public class AuthConfig {
+
+    @Bean
+    public AuthTokenGenerator authTokenGenerator(
+        @Value("${s2s.secret}") String secret,
+        @Value("${s2s.name}") String name,
```

Comment:
is `application.yaml` file change to include these properties missing in this pr?

Reply:
`S2S_SECRET` and `S2S_NAME` come as environment properties (set in Terraform). Should I add them, anyway, so that application.yaml shows the full set of configuration properties?

Reply:
not necessarily in this pr, maybe in a separate PR